src/actions/LogActions.tsx

```diff
@@ -103,17 +121,15 @@ export function showClearLogsModal(): ThunkAction<Promise<void>> {
     )).catch(showError)
   }
 }
-
-export function clearAllLogs(): ThunkAction<Promise<void>> {
-  return async (_dispatch, _getState) => {
-    await clearLogs('activity')
-    await logWithType('activity', 'CLEARED ACTIVITY LOGS')
-    await clearLogs('info')
-    await logWithType('info', 'CLEARED INFO LOGS')
+async function clearAllLogs(): Promise<void> {
+  try {
+    await Promise.all([clearLogs('activity'), logWithType('activity', 'CLEARED ACTIVITY LOGS'), clearLogs('info'), logWithType('info', 'CLEARED INFO LOGS')])
+  } catch (error) {
+    console.error('Error while clearing logs:', error)
```

Comment:
Please put back the `await` chain. While using `Promise.all` lets us do work in parallel for better performance, these tasks can't be done at the same time. If you write something to a file that's being deleted at the same time, it's a race which one will get there first, and you may not log your message at all.

Turning this thunk into a normal function is fine, since we don't actually use the `dispatch` anywhere inside.